src/components/Text/Text.tsx

```diff
@@ -13,7 +13,7 @@ export interface TextProps extends TypographyProps {
 
 const Text: VibeComponent<TextProps, HTMLElement> = forwardRef(
   ({ className, size = "medium", weight = "normal", ellipsis, paragraph, ...typographyProps }, ref) => {
-    const overrideEllipsis = ellipsis || !paragraph;
+    const overrideEllipsis = ellipsis === undefined ? !paragraph : ellipsis;
```

Comment:
ellipsis ?? !paragraph

Reply:
```suggestion
    const overrideEllipsis = ellipsis ?? !paragraph;
```
made sure with Jebeta 😅 it works the same